appcircle_ios_azure_bot_for_swiftlint/1.0.2/component.yaml

```diff
@@ -0,0 +1,64 @@
+platform: iOS
+buildPlatform:
+displayName: Azure DevOps Bot for Swiftlint Report
+description: "Bot script that allows Swiftlint outputs to make changes to Azure DevOps Pull Requests."
+docsUrl: https://docs.appcircle.io/integrations/azure-bot-for-swiftlint-and-detekt#azure-devops-bot-for-swiftlint
+requiredComponents: "appcircle_swiftlint"
+precedingComponents: 
+followingComponents: "appcircle_export_build_artifacts"
```

Comment:
> @boztopuz Since this review is very repetitive, I will not give the same feedback in the rest, but can you provide the checks for all of them?

I mentioned the above situation in the previous review, but since the `export_build_artifacts` for which I did not add a comment were not corrected, I am adding them again for each one.
=> There is no any dependencies with `appcircle_export_build_artifacts`